.gitignore

```diff
@@ -11,3 +11,5 @@ archive.tar.gz
 *.cjsx.js
 /phantomjsdriver.log
 */test/some-tests.*
+*.idea
+package-lock.json
```

Comment:
`package-lock.json` and int's changes should be checked into git.  We haven't firmly moved to npm 5 yet, we're moving in that direction.

Reply:
Oh, I didn't know what the file was for. I didn't see it when I cloned things down so I thought it wasn't supposed to be there.